src/StackAdmin/Azs.Compute.Admin/Module/Azs.Compute.Admin/Generated.PowerShell.Commands/SwaggerPathCommands/Remove-AzsComputeQuota.ps1

```diff
@@ -31,11 +31,6 @@ Changes may cause incorrect behavior and will be lost if the code is regenerated
 
     Remove a compute quota given all the parameters.
 
-.EXAMPLE
```

Comment:
Why did you remove the example?

Reply:
bcoz the same example was pasted twice, i removed once